README.md

```diff
@@ -1 +1,31 @@
-# som_timer_be
\ No newline at end of file
+# som_timer_be
+
+## Endpoints
+
+#### GET  '/api/timers/{timer_id}
+
+```
+#RESPONSE
+{
+    "id": 1,
+    "work_interval": "25:00",
+    "rest_interval": "5:00"
+}
+```
+
+#### PUT  '/api/timers/{timer_id}
```

Comment:
Awesome, I've never actually created a PUT request so excited to put that into practice!

Reply:
Rachel I really appreciate how you're leaning in to the puns.

Reply:
haha that one was unintentional

Reply:
Punintentional